packages/itwinui-css/src/surface/surface.scss

```diff
@@ -18,9 +18,36 @@
   border-radius: var(--iui-surface-border-radius);
   box-shadow: var(--iui-surface-elevation);
   color: var(--iui-color-text);
+
+  &:where([data-iui-layout='true']) {
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+$surface-padding: var(--iui-size-2xs);
+
+@mixin iui-surface-header {
+  display: flex;
+  flex-shrink: 0;
+  align-items: center;
+  overflow: hidden;
+  border-bottom: 1px solid var(--iui-surface-border-color);
+  padding: $surface-padding;
+  scrollbar-gutter: stable both-edges;
 }
 
-// Public facing surface component.
-@mixin iui-surface-component {
-  @include iui-surface;
+@mixin iui-surface-body {
+  flex-grow: 1;
+  overflow-x: hidden;
+  overflow-y: auto;
+
+  [data-iui-padded='false'] & {
```

Comment:
I noticed in the css that `data-iui-padded` only affects `iui-surface-body`, but the demo pages have this data attribute applied to `iui-surface`. Should the data attribute just apply to `iui-surface-body`?

Reply:
@FlyersPh9 Maybe move this attribute to the body? i.e. `&[data-iui-padded='false']`

Reply:
Done, I just pushed this change.